Detectors/TOF/base/include/TOFBase/Digit.h

```diff
@@ -41,15 +47,28 @@ class Digit : public o2::dataformats::TimeStamp<double>
   Int_t getBC() const { return mBC; }
   void setBC(Int_t bc) { mBC = bc; }
 
+  Int_t getLabel() const { return mLabel; }
+  void setLabel(Int_t label) { mLabel = label; }
+
   void printStream(std::ostream &stream) const;
 
-private:
+  void merge(Double_t time, Int_t tdc, Int_t tot);
+
+  void getPhiAndEtaIndex(int& phi, int& eta);
+
+  Bool_t isUsedInCluster() const { return mIsUsedInCluster; }
+
+  void setIsUsedInCluster() { mIsUsedInCluster = kTRUE; }
+
+ private:
   friend class boost::serialization::access;
 
   Int_t mChannel;       ///< TOF channel index
   Int_t mTDC;           ///< TDC bin number
   Int_t mTOT;           ///< TOT bin number
   Int_t mBC;            ///< Bunch Crossing
+  Int_t mLabel;         ///< Index of the corresponding entry in the MC label array
+  Bool_t mIsUsedInCluster; //!/< flag to declare that the digit was used to build a cluster
```

Comment:
We don't write the variable mIsUsedInCluster, so it should be fine like this.

Reply:
We don't write mIsUsedInCluster, so this can stay as it is. 
mChannel can be up to ~157000, so we need an Int. 
mLabel too, cannot be shorter than an Int.
For the rest, we will check and update the comment.